package.json

```diff
@@ -21,7 +21,7 @@
   },
   "homepage": "https://github.com/at-internship/at-sce-ui#readme",
   "dependencies": {
-    "axios": "^0.20.0",
+    "axios": "^0.21.1",
```

Comment:
Why did you update the axios version?

Reply:
It's because I reinstalled the modules, I suppose it's the latest version of axios

Reply:
That is not necessary

Reply:
should be axios": "^0.20.0",

Reply:
Done